Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/PrefabFocusHandler.cpp

```diff
@@ -424,22 +431,17 @@ namespace AzToolsFramework::Prefab
         // closest to the focused instance.
         if (!climbUpResult.m_climbedInstances.empty())
         {
-            AZStd::string prefix = "";
-
-            // Skip the instance closest to the focused instance.
-            auto startInstanceIter = ++climbUpResult.m_climbedInstances.rbegin();
-            for (auto instanceIter = startInstanceIter; instanceIter != climbUpResult.m_climbedInstances.rend(); ++instanceIter)
-            {
-                prefix.append(PrefabDomUtils::PathStartingWithInstances);
-                prefix.append((*instanceIter)->GetInstanceAlias());
-            }
+            // Skip the instance closest to the target instance.
+            AZStd::string prefix = PrefabInstanceUtils::GetRelativePathFromClimbedInstances(
+                ++climbUpResult.m_climbedInstances.crbegin(), climbUpResult.m_climbedInstances.crend());
 
-            m_instanceToTemplateInterface->AppendEntityAliasToPatchPaths(providedPatch, entityId, AZStd::move(prefix));
+            AZStd::string entityAliasPath = m_instanceToTemplateInterface->GenerateEntityAliasPath(entityId, AZStd::move(prefix));
+            m_instanceToTemplateInterface->AppendEntityAliasPathToPatchPaths(providedPatch, AZStd::move(entityAliasPath));
```

Comment:
nit: You can avoid calling the same interface twice by calling AppendEntityAliasToPatchPaths() instead.

Reply:
Fixed.